scss/detail.scss

```diff
@@ -12,14 +17,19 @@
   margin-right: 52px;
 }
 
+.book-info {
+  width: 100%;
+  max-width: 520px;
```

Comment:
There's no need to set a fixed width here. Your book-container already has a width. Then, you have 2 elements: book-cover and book-info. 

With that in mind if you already have a width set in your book cover then the remaining space will be filled by your book-info. So, you can easily just make `width: 100%`